CHANGELOG.md

```diff
@@ -100,6 +100,13 @@ instances according to the [policy configuration](https://github.com/nasa/cumulu
 
 ### Changed
 
+- **CUMULUS-2837**
```

Comment:
Will need to move after merging in latest from `master`

Reply:
f6a0f88a2